File: src/easy/Problem_1528.java
package easy;

public class Problem_1528 {
    public String restoreString(String s, int[] indices) {
        char[] array = new char[s.length()];
        String restoredString = "";

        for (int i = 0; i < indices.length; i++) {
            array[indices[i]] = s.charAt(i);
        }

        for (char c : array) {
            restoredString += c;
        }

        return restoredString;
    }
}

File: c-sharp-solves/Problem_1528.cs
public class Solution
{
    public string RestoreString(string s, int[] indices)
    {
        string restoredString = "";
        char[] stringArray = new char[s.Length];
        for (int i = 0; i < indices.Length; i++)
        {
            stringArray[indices[i]] = s[i];
        }

        foreach (var letter in stringArray)
        {
            restoredString += letter;
        }

        return restoredString;
    }
}